packages/app/pages/course/[cid]/insights.tsx

```diff
@@ -72,22 +71,47 @@ export default function Insights(): ReactElement {
     <>
       <StandardPageContainer>
         <NavBar courseId={Number(cid)} />
-        <InsightContainer>
-          <h1 style={{ margin: "2px" }}>Insights Dashboard - Alpha</h1>
-
-          <div style={{ maxWidth: "200 px" }}>
-            <b style={{ display: "inline-block", marginRight: "12px" }}>
-              Date Range
-            </b>
-            <RangePicker
-              onChange={(_, dateString) =>
-                setDateRange({ start: dateString[0], end: dateString[1] })
-              }
-            />
-          </div>
-        </InsightContainer>
-        <Divider style={{ margin: "0 0 8px 0" }} />
-
+        <Row
```

Comment:
I don't really know our exacts on styled components vs inline styles, maybe these should be styled components?? Not sure

Reply:
Stanley and I originally tried styled components then decided on using the ant-design row cause turned out to work better